1.8/usage/managing-services/expose-service.md

```diff
@@ -1,12 +1,11 @@
 ---
-post_title: Deploying Public Services
-nav_title: Public Services
-menu_order: 10
+post_title: Exposing a Service
+menu_order: 005
 ---
 
-DC/OS agent nodes can be designated as [public](/docs/1.8/overview/concepts/#public) or [private](/docs/1.8/overview/concepts/#private) during [installation](/docs/1.8/administration/installing/). Public agent nodes provide public access to your DC/OS applications. By default apps are launched on private agent nodes. 
+DC/OS agent nodes can be designated as [public](/docs/1.8/overview/concepts/#public) or [private](/docs/1.8/overview/concepts/#private) during [installation](/docs/1.8/administration/installing/). Public agent nodes provide access from outside of the cluster via infrastructure networking to your DC/OS services. By default apps are launched on private agent nodes and are not accessible. 
 
-To launch an app on a public node, you must create a Marathon app definition with the `"acceptedResourceRoles":["slave_public"]` parameter specified.
+To launch an app on a public node, you must create a Marathon app definition with the `"acceptedResourceRoles":["slave_public"]` parameter specified and configure an edge load balancer and service discovery mechanism. This example 
```

Comment:
Suggest: "To launch a service on a public node, create a Marathon app definition..."